CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1 @@
-
-
+Fix: broken email action
```

Comment:
As far as I remember in the case of hotfixes on release/x.y.z branch, we don't use CNR, but directly adding a new entry for 1.12.1 in cep.spec `%changelog` section.

In addition, the version number in package.json shoul move to 1.12.1

Reply:
Done in 
https://github.com/telefonicaid/perseo-fe/pull/434/commits/f756da637066ef70a9812e7cfca5f8e63fb7cade
https://github.com/telefonicaid/perseo-fe/pull/434/commits/6f3d52ef0593ea1d73dd56b402e8fabd98513616
https://github.com/telefonicaid/perseo-fe/pull/434/commits/93241d86e88501e701f99ff0d8600f9d8332826f